Program.cs

```diff
@@ -620,6 +644,11 @@ public void itemShop()
             
         }
 
+        private void morseCodeMessage()
+        {
+            Console.Beep(100, 100);
+        }
+
```

Comment:
This method is never used

Reply:
I start using this method for something else